frontend/src/lib/websocket.ts

```
@@ -0,0 +1,172 @@
+      this.pendingUnsubscriptions.forEach(msg => {
+        this.send(msg);
+      });
+      this.pendingUnsubscriptions = [];
+    };
+
+    this.ws.onmessage = (event) => {
+      try {
+        const msg: WSMessage = JSON.parse(event.data);
+        const typeHandlers = this.handlers.get(msg.type);
+        if (typeHandlers) {
+          typeHandlers.forEach((handler) => handler(msg.payload || {}));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)